### PR TITLE
[4120] block with widgets width fix

### PR DESCRIPTION
**Pull request by Mohamed-halim-sw**
**Related issue(s):**
* Fixes scandipwa/scandipwa/issues/4120

**Problem:**
* Product list widget width fix

**In this PR:**
* CmsPage.style.scss 
      * removed margin from line 118.

### CODE REVIEW

**Comment by Elena-Evsa** [comment]
1. Block with New and recently viewed products are aligned, cards from catalog product list widget is not aligned

image

**Comment by Elena-Evsa** [comment]
1. Passed
2. Same situation on About us page: widget (New products or Category) should be sized with content on any CMS page
image

**Comment by aleksandrakorolova** [comment]
2. Fixed
3. PLP doesn't have padding on the left and right side
Screenshot 2022-02-07 at 11 07 48

**Comment by aleksandrakorolova** [comment]
2. Please, use the same padding-inline as on other pages (14px)
Screenshot 2022-02-21 at 11 01 02

File: packages/scandipwa/src/component/Html/Html.component.js [hunk 1]
```diff
@@ -255,7 +255,6 @@ export class Html extends PureComponent {
         if (!Number.isNaN(Number(script))) {
             document.head.appendChild(script);
         }
-
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/Html/Html.component.js:258]: No need to remove this white space.

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]
See comments

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**